package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
+    "chart.js": "2.7.1",
```

Comment:
You should also have `react-chartjs-2` listed as a dependency. Looks like it got lost when you merged master into this branch. Run `npm install --save react-chartjs-2` to fix this problem.